README.md

```diff
@@ -2,65 +2,121 @@
 
 Warns and then closes issues and PRs that have had no activity for a specified amount of time.
 
-### Usage
+Based on [`actions/stale`](https://github.com/actions/stale).
 
-See [action.yml](./action.yml) For comprehensive list of options.
- 
-Basic:
-```
-name: "Close stale issues"
+Additional features:
+
+- Multiple exempt labels
+- Removes the stale label after activity in the marked issue
+- Add a message when closing an issue
+- DRY run (only logging, no actions)
+- Two outputs (`blocks` and `queue`) which can be used for a Slack action or passed to your own action. These outputs list the closed issues after each run
+
+## Usage
+
+### Arguments
+
+| Argument               | Example                           | Required | Description                                                                                               |
+|------------------------|-----------------------------------|----------|-----------------------------------------------------------------------------------------------------------|
+| GITHUB_TOKEN           | -                                 | Required | Available via environment variables                                                                       |
+| DRY_RUN                | true                              | Optional | Default: false. Execute the commands only to the console. No GitHub actions or Slack messages will be run |
```

Comment:
```suggestion
| DRY_RUN                | true                              | Optional | Default: false. Execute the commands only to the console. No GitHub actions will be run |
```
slack messages was refactored to be used with slack github action so this no longer apply